Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -150,6 +163,71 @@ void CompleteMolQueries(RWMol *mol, int magicVal = 0xDEADBEEF) {
   }
 }
 
+bool startsWith(const std::string &haystack, const char *needle, size_t size) {
```

Comment:
Probably not part of this review, but shouldn't these be in an unnamed namespace not in namespace FileParserUtils?